matrix_client/api.py

```diff
@@ -238,6 +238,16 @@ def send_message(self, room_id, text_content, msgtype="m.text"):
             self.get_text_body(text_content, msgtype)
         )
 
+    def send_html(self, room_id, html_content, msgtype="m.text"):
+        """Send an html formatted message.
+
+        Args:
+            room_id (str): The room ID to send the event in.
+            html_content (str): The html formatted message to be sent.
+        """
+        return self.send_message_event(room_id, "m.room.message",
+                                       self.get_html_body(html_content, msgtype))
```

Comment:
> I'd prefer to have send_html in room.py have an optional parameter for body which would either be filled in by the developer, or generated by our little function here.

This is pretty confusing since `body` and `html_content` should be mutually exclusive, but what happens if  `send_html` is called with both parameters?